tests/testcontainers-env/pom.xml

```diff
@@ -45,10 +45,20 @@
             <artifactId>mysql</artifactId>
             <version>${dependency.testcontainers.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>mariadb</artifactId>
+            <version>${dependency.testcontainers.version}</version>
+        </dependency>
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>elasticsearch</artifactId>
             <version>${dependency.testcontainers.version}</version>
         </dependency>
+
+        <dependency>
```

Comment:
please check the location, couldn't set the scope of dependency as test, but since it's located in tests only should be fine

Reply:
I think we should keep additional drivers in a `test` scope. What do you think of moving these drivers to the `tests/tas-elasticsearch/pom.xml`.